packages/supercolliderjs/src/index.d.ts

```
@@ -0,0 +1,28 @@
+    server: typeof ServerPlus;
+  };
+  dryads: typeof dryads;
+  lang: typeof langLib;
+  map: typeof map;
+  msg: typeof msg;
+  SCLangError: typeof SCLangError;
+  resolveOptions: typeof resolveOptions;
+};
+
+export default supercolliderjs;
+export const server: typeof supercolliderjs.server;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'server'. Consider to delete it or rename it.

'server' is defined here:


(lint/suspicious/noRedeclare)